src/views/Farms/components/Tooltip/Tooltip.tsx

```diff
@@ -44,11 +45,11 @@ const Container = styled.div`
   }
 `
 
-const Tooltip: React.FunctionComponent<TooltipProps> = ({ content, children }) => {
+const Tooltip: React.FunctionComponent<TooltipProps> = ({ content, children, className }) => {
   return (
     <Container>
       {children}
-      <TooltipContent>{content}</TooltipContent>
+      <TooltipContent className={className}>{content}</TooltipContent>
```

Comment:
Required to allow tooltip to be restyled here https://github.com/pancakeswap/pancake-frontend/pull/773/files#diff-7139049a58c241ef6622e4a8f989946a026906be596a4b442f86a423d493a436R6.

By default it sits above the component, hence the restyling. 

I think work is already being done on this tooltip - hence not going further and making accept a 'position' prop and dynamically style it that way or similar